nix/not-os-image.nix

```diff
@@ -34,6 +40,17 @@ let
         ip addr add 10.0.2.15/24 dev eth0
       '';
 
+      boot.initrd.availableKernelModules = [ "virtio_console" "virtio_mmio" ];
+      # to activate at boot time: boot.initrd.kernelModules = [ "virtio_mmio" ];
+      boot.kernelPatches = [ {
+        name = "virtio-mmio-cmdline";
+        patch = null;
+        extraConfig = ''
+                VIRTIO_MMIO_CMDLINE_DEVICES y
```

Comment:
I could add a cache, but I think I could also try just to build a single kernel module with this option set. I checked the code: It only has effect within the kernel module.

Reply:
I will do this after merge.